keyboards/rainkeebs/plop/info.json

```
@@ -0,0 +1,66 @@
+{
+    "manufacturer": "rainkeebs",
+    "keyboard_name": "plop",
+    "maintainer": "rainkeebs",
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7070",
+        "vid": "0x726B"
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x12": {
```

```suggestion
        "LAYOUT": {
```
\+ whatever other fixes this entails.